### PR TITLE
Apply consistent arch name "arm64"

### PR DESCRIPTION
**Pull request by kiwik**
1. Change job metadata from aarch64 to arm64
2. Remove useless nodeset "ubuntu-bionic-vxh-arm64"

Related-Bug: theopenlab/openlab#318

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -2173,7 +2173,7 @@
       - Project:apache/hbase
       - Application:Hbase@master
       - OS:ubuntu-xenial
-      - Arch:aarch64
+      - Arch:arm64
       - BuildType:Compile
```
Comment (bzhaoopenstack) [inline zuul.d/jobs.yaml]: looks duplicated with https://github.com/theopenlab/openlab-zuul-jobs/pull/588

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -2189,5 +2189,5 @@
       - Project:apache/hive
       - Application:Hive@master
       - OS:ubuntu-xenial
-      - Arch:aarch64
+      - Arch:arm64
```
Comment (bzhaoopenstack) [inline zuul.d/jobs.yaml]: And here, looks duplicated with https://github.com/theopenlab/openlab-zuul-jobs/pull/588

File: zuul.d/jobs.yaml (hunk 1 above)
Comment (huangtianhua) [reply zuul.d/jobs.yaml]: Maybe we remain 'aarch64' for Arch metadata? We specify the 'Arch' comes from uname -a in redesign doc, or modify the doc to just tell we use arm64?

Comment (kiwik) [reply zuul.d/jobs.yaml]: @huangtianhua I have updated all of the related documents, include: redesign and docs.openlabtesting.org, please doube check it, thank you. And I think `Arch:arm64` is clear enough.

File: zuul.d/jobs.yaml (hunk 2 above)
Comment (kiwik) [reply zuul.d/jobs.yaml]: Yep, I will rebase if #588 is merge first.

File: zuul.d/jobs.yaml (hunk 1 above)
Comment (huangtianhua) [reply zuul.d/jobs.yaml]: OK, let's set it to arm64, and I will change related in my spark job.

**Pull request merged**